client/my-sites/upgrades/domain-management/edit/mapped-domain.jsx

```diff
@@ -110,10 +111,11 @@ const MappedDomain = React.createClass( {
 
 		return (
 			<VerticalNavItem path={ path }>
-				{ this.translate( 'DNS Records' ) }
+				{ this.props.translate( 'DNS Records' ) }
 			</VerticalNavItem>
 		);
 	}
 } );
 
-module.exports = MappedDomain;
+export { MappedDomain };
```

Comment:
If it extended the class, you could one line it. `export const MappedDomain`. Easier to get along with I guess. :)